src/styles/typography.scss

```diff
@@ -148,11 +158,12 @@ $small-empty-state-height: 12px;
   font-size: $font-size-input;
   font-weight: $font-weight-normal;
   line-height: 22px;
+
   @include theme-prop(color, primary-text-color);
 }
 
 @mixin line-clamp($lines: 1) {
-  display: -webkit-box;
+  display: box;
```

Comment:
this is a breaking change - there is no `display: box` property

Reply:
yes this is not good. I've ignored these changes for now here:
https://github.com/mondaycom/monday-ui-react-core/pull/310/files